backend/graphql_api/utils.py

```diff
@@ -16,6 +16,6 @@ def resolve_localized_field(field_name: str):
 
     def _resolve(parent, info, lang: str = DEFAULT_LANGUAGE):
         messages = getattr(parent, field_name)
-        get_message_in_language(messages, lang)
+        return get_message_in_language(messages, lang)
```

Comment:
I could say something about those rtype annotations in this instance 😁

Reply:
Yes and no. Whatever it returns is actually dependent on `getattr(parent, field_name)` which may or may not be a `dict[str, str]`. `-> Any` can also be `None`.